Formula/stubby.rb

```diff
@@ -56,22 +59,26 @@ def plist; <<-EOS.undent
   end
 
   test do
-    (testpath/"stubby_test.conf").write <<-EOS.undent
-      { resolution_type: GETDNS_RESOLUTION_STUB
-      , dns_transport_list: [ GETDNS_TRANSPORT_TLS, GETDNS_TRANSPORT_UDP, GETDNS_TRANSPORT_TCP ]
-      , listen_addresses: [ 127.0.0.1@5553]
-      , idle_timeout: 0
-      , upstream_recursive_servers:
-        [ { address_data: 145.100.185.15},
-          { address_data: 145.100.185.16},
-          { address_data: 185.49.141.37}
-        ]
-      }
+    output = shell_output("ls #{etc}/stubby/")
```

Comment:
Why the check for this?

Reply:
The update from 0.1.2 to 0.1.3 broke new installs because the stubby.yml file was not installed and the stubby service needed it. This check was added to make sure that doesn't happen again just in case the configuration is ever changed in future. Having stubby read the stubby.yml file directly currently requires root privileges (because it causes stubby to bind to port 53) so it isn't done in the test. We plan to add a separate option to just parse the config file without acting on it in a future release and will update the test at that point, but this seemed the next best way to ensure it was present.

Reply:
This sounds a bit paranoid in terms of the likelihood of the same regression occurring again.

But
```
    assert_predicate etc/"stubby/stubby.yml", :exist?
```

Reply:
It is totally paranoid, my concern is that in certain cases a lack of config file can cause stubby to fallback to sending DNS queries in clear text without the user being aware if they don't look in the logs. I'm working on a couple of command line options for 0.1.4 that will make the installation easier to test directly. 

Thanks for the suggestion - update on the way.